src/main/java/com/groupthree/mancala/HelloApplication.java

```diff
@@ -15,9 +15,24 @@ public void start(Stage stage) throws IOException {
         stage.setTitle("Group 3");
         stage.setScene(scene);
         stage.show();
+        //startScreen(stage);
+        //playerDashboardScreen(stage);
+    }
+public void startScreen(Stage stage) throws IOException {
+        FXMLLoader fxmlLoader = new FXMLLoader(HelloApplication.class.getResource("start-screen-view.fxml"));
+        Scene scene = new Scene(fxmlLoader.load());
+        stage.setTitle("Group 3");
+        stage.setScene(scene);
+        stage.show();
+    }
+    public void playerDashboardScreen(Stage stage) throws IOException {
+        FXMLLoader fxmlLoader = new FXMLLoader(HelloApplication.class.getResource("player-dashboard-view.fxml"));
```

Comment:
The methods to show the screens are not necessary for the Main class, so please remove them.
Ideally, the methods will be called from within other screens' controllers, and not the main one.

Reply:
@mofe64, Thanks for the review. Treated